src/components/TransactionList/TransactionRow/TransactionRowTs.ts

```diff
@@ -44,6 +57,9 @@ import { DateTimeFormatter } from '@js-joda/core';
         explorerBaseUrl: 'app/explorerUrl',
         networkConfiguration: 'network/networkConfiguration',
         currentProfile: 'profile/currentProfile',
+        currentAccount: 'account/currentAccount',
+        currentAccountMultisigInfo: 'account/currentAccountMultisigInfo',
+        generationHash: 'network/generationHash',
```

Comment:
I did find `generationHash` use on this code, do we need it?

Reply:
@AnthonyLaw  it is from the `network/generationHash` network store.

Reply:
yeah. I mean `generationHash` is not using anywhere on `TransactionRowTs.ts`